securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -54,6 +54,29 @@ def _journalist_logs_in(self):
         headline = self.driver.find_element_by_css_selector('span.headline')
         assert 'Sources' in headline.text
 
+    def _journalist_visits_col(self):
+        self.driver.find_element_by_css_selector(
+            '#un-starred-source-link-1').click()
+
+    def _journalist_select_first_doc(self):
+        self.driver.find_elements_by_name('doc_names_selected')[0].click()
+
+    def _journalist_delete_one_javascript(self):
+        self.driver.find_element_by_id('delete_selected').click()
+        self._alert_wait()
+
+    def _journalist_verify_delete_one_javascript(self):
+        self._journalist_select_first_doc()
+        self._journalist_delete_one_javascript()
+        self._alert_dismiss()
+        selected_count = len(self.driver.find_elements_by_name(
+            'doc_names_selected'))
+        assert selected_count > 0
+        self._journalist_delete_one_javascript()
+        self._alert_accept()
+        assert selected_count > len(self.driver.find_elements_by_name(
+            'doc_names_selected'))
```

Comment:
Are these new journalist steps being used anywhere? They could extend an existing functional flow, e.g. the `test_submit_and_retrieve_happy_path` or they could be a separate new functional test, which might be a little clearer:

```
    def test_javascript_alerts_on_journalist_interface(self):
        self._source_visits_source_homepage()
        self._source_chooses_to_submit_documents()
        self._source_continues_to_submit_page()
        self._source_submits_a_message()
        self._source_logs_out()
        self._journalist_logs_in()
        self._journalist_visits_col()
        self._journalist_verify_delete_one_javascript()
```

(Either approach is fine, just want to make sure these steps are being used in the tests)